action.yml

```diff
@@ -35,20 +35,6 @@ runs:
           echo "The action doesn't support building on Windows. Exiting script."
           exit 1
         fi
-    - name: Install Homebrew
-      shell: bash
-      run: |
-        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-        if [[ "$OSTYPE" == "darwin"* ]]; then
-          echo 'export PATH="/usr/local/bin:$PATH"' >> $GITHUB_PATH
-        elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
-          echo "export PATH=\$PATH:/home/linuxbrew/.linuxbrew/bin" >> $GITHUB_PATH
-        fi
-    - name: Install dependencies
-      shell: bash
-      run: |
-        brew update
-        brew install wget jq
```

Comment:
AFAIK, `jq` does not come pre-installed in macOS or Linux, so we must install `jq`.

Reply:
As far as I can see, we need to JQ when only targeted to `latest` or `current`. There doesn't seem to be any need to activate Brew processes for this right away. I'm configured dependencies to load when they are needed. So they won't slow down the workflow when not needed.